flashflood/__init__.py

```diff
@@ -69,15 +69,30 @@ def put(self, data, event_id: str=None, date: datetime=None):
         self._upload_journal(_Journal(journal_id, manifest, io.BytesIO(data)), is_new=True)
         return Event(event_id, date, data)
 
-    def journal(self, number_of_events=10):
+    def journal(self, minimum_number_of_events=100, minimum_size=None):
```

Comment:
Readme still says `ff.journal(number_of_events=5000)`.

Also, I'd consider adding typing, i.e. (` def journal(self, minimum_number_of_events: int =100, minimum_size: int = None)`), both here and elsewhere.

Reply:
`README.md` updated.

I'll consider adding type hints in a future PR.